card/config/initializers/01_core_extensions/array.rb

```diff
@@ -2,7 +2,7 @@ module CoreExtensions
   module Array
     def to_pointer_content
       map do |item|
-        "[[#{item}]]"
+        item =~ /^\[\[.+\]\]$/ ? item : "[[#{item}]]"
       end.join "\n"
     end
```

Comment:
depends on what you feed in. Instead of checking the input where I use it I wanted to make it work for any kind of input.